sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -1,3 +1,8 @@
+"""We can not use pytest here, because we run
+build_tools/azure/test_pytest_soft_dependency.sh on these
+tests to make sure estimator_checks works without pytest.
+"""
```

Comment:
Let's use traditional python comments rather than a module level docstring as this is not API documentation for the module (for module users), but rather a note to developers.

```suggestion
# We can not use pytest here, because we run
# build_tools/azure/test_pytest_soft_dependency.sh on these
# tests to make sure estimator_checks works without pytest.
```